src/Tools/dotnet-monitor/Egress/Extension/EgressExtension.cs

```diff
@@ -158,5 +191,34 @@ private void ValidateFileExists(FileInfo fileInfo)
                 ExtensionException.ThrowFileNotFound(_manifest.Name, fileInfo.FullName);
             }
         }
+
+        private Dictionary<string, string> GetConfigurationSection(string providerName, string providerType)
+        {
+            IConfigurationSection providerTypeSection = _configurationProvider.GetProviderTypeConfigurationSection(providerType);
+            IConfigurationSection providerNameSection = providerTypeSection.GetSection(providerName);
```

Comment:
I'm assuming this was just a 1:1 copy-paste of the existing implementation (with the refactored method name for `GetProvider**Type**ConfigurationSection`), but isn't this effectively recreating what `GetProviderConfigurationSection` does?

Reply:
Will fix this in next iteration